### PR TITLE
feat : Wishlist Page Added

### PR DESCRIPTION
**Pull request by vikrantkode**
# Changes Done
    Wishlist Page Added
    Can add to wishlist from product page and cart page
    Can remove wishlist items from  wishlist page

### CODE REVIEW

File: src/context/Wishlist-Context.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+const addToCartClickHandler = async (product, setItemsInCart, encodedToken, navigate) => {
+   
+    if(encodedToken){
+      try {
+        const cartItems = await axios.post( `/api/user/cart`, {product} ,
+          { headers: { authorization: encodedToken } }
+        );
+        setItemsInCart(cartItems.data.cart)
+      } catch (err) {
+        alert("error from server", err)
```
Comment (Asuraking1n) [inline src/context/Wishlist-Context.jsx:14]: For error use toasts, it was suggested by a MA in a code review session

**Review by Asuraking1n — COMMENTED** [review]
Even after login, login button is visible in navbar instead of logout

File: src/components/ProductCard/ProductCard.jsx [hunk 2]
```diff
@@ -1,35 +1,48 @@
...
+    //   if(encodedToken){
+    //     try {
+    //       const cartItems = await axios.post( `/api/user/cart`, {product} ,
+    //         { headers: { authorization: encodedToken } }
+    //       );
+    //       setItemsInCart(cartItems.data.cart)
+    //     } catch (err) {
+    //       alert("error from server", err)
+    //     }  
+    //   } else {
```
Comment (sakshi006) [inline src/components/ProductCard/ProductCard.jsx]: Can remove commented code

**Review by sakshi006 — APPROVED** [review]
Looks good to merge

File: src/components/ProductCard/ProductCard.jsx (hunk 2 above)
Comment (vikrantkode) [reply src/components/ProductCard/ProductCard.jsx]: I removed it

File: src/context/Wishlist-Context.jsx (hunk 1 above)
Comment (vikrantkode) [reply src/context/Wishlist-Context.jsx:14]: yeah will add after finishing  it.

**Pull request merged**